sample-apps/python/django_frontend_service/frontend_service_app/views.py

```diff
@@ -105,3 +107,18 @@ def get_xray_trace_id():
     xray_trace_id = f"1-{trace_id[:8]}-{trace_id[8:]}"
 
     return JsonResponse({"traceId": xray_trace_id})
+
+def mysql(request):
+    logger.info("mysql received")
+    try:
+        connection = pymysql.connect(host=os.environ["RDS_MYSQL_CLUSTER_ENDPOINT"],
+                                     user=os.environ["RDS_MYSQL_CLUSTER_USERNAME"],
+                                     password=os.environ["RDS_MYSQL_CLUSTER_PASSWORD"],
+                                     database=os.environ["RDS_MYSQL_CLUSTER_DATABASE"])
+        with connection:
+            with connection.cursor() as cursor:
+                cursor.execute("SELECT * FROM tables LIMIT 1;")
+                return get_xray_trace_id()
+    except Exception as exception:  # pylint: disable=broad-except
+        logger.exception("Exception Occurred")
+        raise exception
```

Comment:
For our other APIs, we `return get_xray_trace_id()` regardless of the outcome of the API call, as we don't assume the trace ID was not generated despite an exception being thrown. It's possible that despite an exception a trace ID is still generated and it's important for us to be able to see if the trace generated by the instrumentation on the application is also appearing in the console.
```suggestion
            with connection.cursor() as cursor:
                cursor.execute("SELECT * FROM tables LIMIT 1;")
                return get_xray_trace_id()
    except Exception as exception:  # pylint: disable=broad-except
        logger.error("Could not complete http request to RDS database:" + str(e))
        return get_xray_trace_id()
```

Reply:
Addressed this comment in the last commits.